{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -127,3 +127,8 @@ APP_DEFAULT_SECURE_HEADERS['content_security_policy']['default-src'].append(
     S3_ENDPOINT_URL
 )
 {%- endif %}
+
+
+# Invenio-Records-Resources
+# =========================
+SERVER_HOSTNAME = "127.0.0.1:5000"
```

Comment:
**Comment:** Just a note on the variable name......I think it's very easy to confuse with Flask's ``SERVER_NAME``.

Reply:
My thinking was to use a similar name because it does what `SERVER_NAME` should do without any further gotchas and it is exactly what it means: the server's hostname. 

We can't use `SERVER_NAME` directly because it breaks the containerized setup. (I wish I had more details but I encountered this in our own local install before starting on invenio-rdm: https://github.com/galterlibrary/InvenioRDM-at-NU/pull/288/files#diff-7bdd48e8822b88393e93ed6650b7eeeeR190 some time ago).

Other names welcome :)

Reply:
``SERVER_NAME`` also have other issues that makes it hard to predict what happens - both with ``url_for`` and other things.

I would go with something like:

- HOSTNAME
- SITE_URL (something we had in Invenio-theme if I remember correctly)
- INSTANCE_HOSTNAME

or some variations over above.